eventmesh-common/src/main/java/org/apache/eventmesh/common/protocol/grpc/protos/BatchMessage.java

```diff
@@ -1555,9 +1555,8 @@ public int getSerializedSize() {
                 (!getTopicBytes().isEmpty() ?
                         com.google.protobuf.GeneratedMessageV3.computeStringSize(3, topic_) : 0);
 
-        for (int i = 0; i < messageItem_.size(); i++) {
-            size += com.google.protobuf.CodedOutputStream
-                    .computeMessageSize(4, messageItem_.get(i));
+        for (MessageItem messageItem : messageItem_) {
```

Comment:
@Orcohen33 suggest use reactor pattern ,use messageItem_.foreach().

Reply:
Hey @jonyangx , I've changed it again and pushed it.